data/prrajput1199.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "Pratik Rajput",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open-Source enthusiastic",
+  "avatar": "https://github.com/prrajput1199.png",
+  "tags": ["Open Source", "react"],
+  "links":[
+  {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/PratikR61035031",
+      "icon": "FaTwitter"
+  },
+  {
+    "name": "Follow me on Github",
+    "url" :"https://github.com/prrajput1199",
+    "icon": "Fagithub"
```

This is an invalid icon. Please update it as suggested! 

```suggestion
    "icon": "FaGithub"
```